dev/doc/changes.md

```diff
@@ -46,9 +46,9 @@ We changed the type of the following functions:
 
 - `Global.body_of_constant`: same as above.
 
-We renamed the following datatypes:
+We have changed the representation of the following types:
 
-- `Pp.std_ppcmds` -> `Pp.t`
```

Comment:
Did you intend to remove this changes entry?

Reply:
Yup as it is subsumed by the more general comment a bit before [we didn't remove but alias]